meetings/2021-06-17.md

```diff
@@ -17,5 +17,121 @@
 * Data model new issues/concerns?
 * Next steps
 
-### Notes
 
+### Present
+- Travis: PM @Microsoft
+- noamh: Ecel online
+- Yego: Flutter, Eng @Google
+- Koji: Lead for text layout @Google
+- Fernando: Tech manager for canvas on Chrome
+- Sayna: PM @Microsoft
+- Sushanth: Eng Mgr @Microsoft
+- Mike: Graphics engineer works on SKIA @Google
+
+### Summary
+Use cases:
+- layout and text flow
+- shaping: a more lower level solution that requires knowledge of data about glyph shapes and geometries.
+
+Concerns about font-files:
```

Comment:
While true, the first two bullets weren't really major points that I would include in the summary :)
Last one is definitely spot-on though.